tests/general/AnnotationBearTest.py

```diff
@@ -0,0 +1,140 @@
+import unittest
+from queue import Queue
+
+from bears.general.AnnotationBear import (find_start_end,
+                                          find_singleline_comments,
+                                          in_ranges,
+                                          calc_line_col,
+                                          AnnotationBear)
+from coalib.results.SourceRange import SourceRange
+from coalib.settings.Section import Section
+from tests.LocalBearTestHelper import execute_bear
+from coalib.settings.Setting import Setting
+
+
+class AnnotationBearTest(unittest.TestCase):
+
+    def setUp(self):
+        self.section = Section("")
+        self.section.append(Setting('language', 'c'))
+        self.section.append(Setting('language_family', 'c'))
+
+    def assertRange(self, file, uut, sourcerange):
+        with execute_bear(uut, "filename", file) as results:
+            print("results", results)
+            for result in results:
```

Comment:
this line checks that no other results are yielded, it was custom for this bear, it could be made generic by including the result in the param